src/OpenSage.Game/Logic/Object/Helpers/StatusDamageHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace OpenSage.Logic.Object.Helpers
+{
+    internal sealed class StatusDamageHelper : ObjectHelperModule
+    {
+        internal override void Load(StatePersister reader)
+        {
+            reader.PersistVersion(1);
+
+            base.Load(reader);
```

Comment:
The pattern we've followed elsewhere I believe is:
```cs
reader.BeginObject("Base");
base.Load(reader);
reader.EndObject();
```
I think that helps nest the json properly when writing saves.

Reply:
Thanks, I missed that one - https://github.com/OpenSAGE/OpenSAGE/commit/c0ecc72697f272d7eaadcb356c3053614de592c3